libr/core/anal_tp.c

```diff
@@ -5,11 +5,16 @@
 #include <r_util.h>
 #include <r_core.h>
 
-static bool r_anal_emul_init (RCore *core) {
+static bool r_anal_emul_init(RCore *core, bool *state) {
+	state[0] = r_config_get_i (core->config, "esil.romem");
```

Comment:
what about using constants instead of immedaites to index that array of booleans? maybe that will make clear the size of it